plugins/modules/change_request.py

```diff
@@ -427,7 +427,7 @@ def main():
             ("state", "authorize", ("assignment_group",)),
             ("state", "scheduled", ("assignment_group",)),
             ("state", "assess", ("assignment_group",)),
-            ("state", "on_hold", ("hold_reason",)),
+            ("on_hold", True, ("hold_reason",)),
```

Comment:
Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example.

Reply:
I've now added the fragment